### ISSUE TITLE
Rapid scroll knob

### ISSUE THREAD

**Issue by Noitarud** [scoute-dich/browser#885]
I have been using an app that shows a knob when the user scrolls the page and would appreciate such a thing in a browser, it moves towards the bottom corner on scrolling that way and the user can grab it and rapidly move over the page.
Perhaps place it on the left if the user is left handed.

**Comment by scoute-dich** [comment]
No new features at the moment. I have not the time, sorry.

**Comment by Noitarud** [comment]
Why did you close this? Wouldn't the idea get forgotten? You may not get another person suggesting this...

**Comment by pc00per** [comment]
@Noitarud Mate. Just use any other browser if you want rich features. There's one called "Via", but it's closed source. 
But once I asked a feature on the dev's telegram chat tagging him, & he immediately replied he'll work on it. And the next day update came with the feature built-in.

I guess whether it's open source or closed source, it just depends on how the dev is, how much time he spends on his projects & how much he gets donations.

**Comment by Noitarud** [comment]
In this case, when you are also the end user, work is less of a chore since you are rewarding yourself, but also in this case, he probably is having a holiday, or craving it, my personal limit working full-on (dawn to bed) was three months. Then also the recent developments in the world… Me, I can program a bit, but only VB (I had looked into others but some howtos are difficult to understand).
@pc00per, It is just closing it kinda appears as snobbing (since it seems it would be forgotten as 'closed', and I will not be always back to attract attention back to it), so thought I needed to try convince, that was all.
